activeEvents.js

```diff
@@ -0,0 +1,34 @@
+var description = null;
+var clearButton = document.querySelector('#clear')
+Carlot = (function(originalFunction) {
+  originalFunction.activeEvents = function() {
+    carContainer = document.querySelectorAll('.car-container');
+    for (let i = 0; i < carContainer.length; i++) {
+      carContainer[i].addEventListener('click', function() {
```

Comment:
Recheck your logic in the cardContainer click handler. When I click a card, any other highlighted cards need to revert to their original state. As it is, I can click a card twice and its background changes back but I can still type into the input to change its text. Plus, selecting a second card doesn't reset the other card(s) back to their original state.